_includes/analytics/facebook_analytics.html

```diff
@@ -0,0 +1,24 @@
+<!-- Facebook Pixel Code -->
+<script>
+  !function(f,b,e,v,n,t,s)
+  {if(f.fbq)return;n=f.fbq=function(){n.callMethod?
+  n.callMethod.apply(n,arguments):n.queue.push(arguments)};
+  if(!f._fbq)f._fbq=n;n.push=n;n.loaded=!0;n.version='2.0';
+  n.queue=[];t=b.createElement(e);t.async=!0;
+  t.src=v;s=b.getElementsByTagName(e)[0];
+  s.parentNode.insertBefore(t,s)}(window, document,'script',
+  'https://connect.facebook.net/en_US/fbevents.js');
+  fbq('init', '{{ site.facebook_analytics_pixel_id }}');
+  fbq('track', 'PageView');
+  {% if page.title == 'Book an Event' %}
```

Comment:
Let's make a note to move this away from a string check in the future, but for now if it's including correctly :shipit: